apps/app/src/app/pages/(components)/components/(data-table)/data-table.preview.ts

```diff
@@ -78,45 +78,47 @@ export type Payment = {
 				</hlm-menu>
 			</ng-template>
 		</div>
-		<div class="border-border mt-4 block h-[18rem] w-full overflow-auto rounded-md border">
+		<div class="overflow-hidden rounded-md border">
 			<!-- we defer the loading of the table, because tanstack manipulates the DOM with flexRender which can cause errors during SSR -->
 			@defer {
-				<table hlmTable class="w-full">
-					<thead hlmTHead>
-						@for (headerGroup of _table.getHeaderGroups(); track headerGroup.id) {
-							<tr hlmTr>
-								@for (header of headerGroup.headers; track header.id) {
-									<th hlmTh [attr.colSpan]="header.colSpan">
-										@if (!header.isPlaceholder) {
-											<ng-container
-												*flexRender="header.column.columnDef.header; props: header.getContext(); let headerText"
-											>
-												<div [innerHTML]="headerText"></div>
+				<div hlmTableContainer>
```

Comment:
The checkbox vertical alignment seems slightly off:

<img width="131" height="60" alt="image" src="https://github.com/user-attachments/assets/853aa051-07e8-45c2-a563-20bba6552385" />

Its a bit high compared to the shadcn example

Reply:
I take a look, thank you!